.eslintrc.yml

```
@@ -0,0 +1,112 @@
+parser: "@babel/eslint-parser"
+
+env:
+  node: true
+  es6: true
+  mocha: true
+  browser: true
+
+plugins:
+  - sonarjs
+
+extends:
+  - eslint:recommended
+  - plugin:sonarjs/recommended
+
+parserOptions:
+  requireConfigFile: false
```

Needed to make it work with async 

```suggestion
  requireConfigFile: false
  ecmaVersion: 2018
```